frontend/app/component/election/ElectionProgress.tsx

```diff
@@ -15,10 +15,18 @@ export function ElectionProgress() {
 
   const stats: Stat[] = useMemo(() => {
     const total = statuses.length;
+    const totalFirstEntry = statuses.filter((s) =>
+      ["second_entry", "second_entry_unfinished", "second_entry_in_progress", "definitive"].includes(s.status),
+    ).length;
```

Comment:
To avoid typo's and have it a bit more explicit I would prefer something like:

```suggestion
    const firstEntryFinished: PollingStationStatus[] = [
      "second_entry",
      "second_entry_unfinished",
      "second_entry_in_progress",
      "definitive",
    ];
    const totalFirstEntry = statuses.filter((s) => firstEntryFinished.includes(s.status)).length;
```